analytics-core/src/main/scala/org/ekstep/analytics/framework/conf/AppConf.scala

```diff
@@ -12,9 +12,13 @@ object AppConf {
     lazy val conf = defaultConf.withFallback(envConf);
 
     def getConfig(key: String): String = {
-        if (conf.hasPath(key))
+        val keyValue = if (conf.hasPath(key)){
```

Comment:
Can this be handled as an Option with getOrElse? Or if it is a scala map, we can directly do a getOrElse if I am right.

Reply:
this was done for some debugging purpose. reverted to the original form